packages/css/src/components/form/form-groups/confirm/form-group-confirm.template.ts

```diff
@@ -0,0 +1,43 @@
+import { FormGroupConfirm } from '@dso-toolkit/sources';
+import { html, nothing, TemplateResult } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+
+import { selectableTemplate } from '../../../selectable/selectable.template';
+
+export function formGroupConfirmTemplate(formGroup: FormGroupConfirm<TemplateResult>): TemplateResult {
+  const errorTextId = `${formGroup.id}-error-text`;
+  const helpTextId = `${formGroup.id}-help-text`;
+
+  const ariaDescribedBy = [
+    formGroup.errorText ? errorTextId : undefined,
+    formGroup.helpText ? helpTextId : undefined
+  ]
+    .filter(s => !!s)
+    .join(' ') || undefined;
+
+  return html`
+    <fieldset
+      class="form-group dso-confirm ${classMap({ 'dso-required': !!formGroup.required, [`dso-${formGroup.state}`]: !!formGroup.state })}"
+      aria-describedby=${ifDefined(ariaDescribedBy)}
+    >
+      <div class="dso-field-container">
+        ${formGroup.selectables.map(selectable =>
```

Comment:
Deze werkform ("Form Groups: confirm") heeft geen ondersteuning voor meerdere selectables, en dat is wat je met deze `.map()` hebt gemaakt. Zie de njk file: https://dso-toolkit.nl/39.0.0/components/detail/group-confirm.html, daar wordt gewoon één selectable aangeroepen.

Reply:
Dat zou wel moeten werken. Ik denk dat het niet werkt omdat je `@dso-toolkit/sources` niet hebt gebuild of de verkeerde volgorde hebt gehanteerd:

* Eerst in `@dso-toolkit/sources` FormGroupConfirm de property `selectables: Selectable<T>[];` naar `selectable: Selectable<T>;` (dus geen array)
* Dan succesvol compilen
* Dan deze file aanpassen naar wat je hierboven stelt

Reply:
Volgens mij is het nu goed.